serve/helpers/delfolder.js

```diff
@@ -0,0 +1,16 @@
+const fse = require('fs-extra');
```

Comment:
Why do we need an extra js  file `deletefolder.js` ? Shouldn't `deleteDir` function go in `folder.js`? I feel all folder related functions should be in one folder.

Reply:
I will put it in the same folder